public/src/inverted-index.js

```diff
@@ -1,19 +1,21 @@
-/** @class representing an Index. */
-class Index {
 /**
-* Constructor initializes indices to an empty object.
+* @class representing an Index
 */
+class Index {
+  /**
+  * Constructor initializes indices to an empty object
+  */
   constructor() {
     this.indices = {};
   }
 
-/**
- * @method fileCheck
- *
- * Checks if file is empty, valid or object
- *
- * @param {object} fileContents
- */
+  /**
+  * @method fileCheck
+  *
+  * Checks if file is empty, valid or object
+  *
+  * @param {object} fileContents
+  */
   fileCheck(fileContents) {
```

Comment:
### Critical

This never actually ensures that the file compulsorily has `title & text` fields, it should